src/nidigital/metadata/functions.py

```diff
@@ -1702,26 +2292,38 @@
     'GetPinResultsPinInformation': {
```

Comment:
Metadata from this function should be copied to https://github.com/ni/nimi-python/blob/30687660fc0bc57b32d6b9639d95f55a52a7a312/src/nidigital/metadata/functions_addon.py#L120

..so that the `get_pin_results_pin_information()` public method docstring is filled out.

Reply:
Done

Reply:
Documentation for return type is still missing.

Reply:
There is a description for the return type that is separate from the description for the function.